File: foo.py
#!/usr/bin/env python
import os, sys, time
import numpy as np
import matplotlib.pyplot as plt

def f0(m=8):
    x = np.linspace(-m, m, 101)
    y = np.cos(x)
    y2 = np.tan(x)
    plt.subplot(211)
    plt.plot(x, y)
    plt.subplot(212)
    plt.plot(x, y2)
    plt.savefig(f"{m}.png")
    return m

if __name__ == "__main__":
    t = time.perf_counter()
    f0(10)
    print(time.perf_counter() - t)

File: foo.jl
#!/usr/bin/env julia
using Plots
unicodeplots()

function f(x, μ=0, δ=1)
    y = 1 / (√(2*pi) * δ) * ℯ^(-(x - μ)^2 / (2 * δ^2))
end

p = plot(f)
display(p)
